watertap/core/plugins/solvers.py

```diff
@@ -39,9 +39,10 @@ def _presolve(self, *args, **kwds):
 
         self._tee = kwds.get("tee", False)
 
-        # Set the IDAES-default tol
+        # Set the default watertap options
         if "tol" not in self.options:
             self.options["tol"] = 1e-08
+        if "constr_viol_tol" not in self.options:
```

Comment:
Same test is failing for the same reason, but the rate of convergence has improved with these changes.

Reply:
Also, the rate of convergence for this test on this branch is significantly worse than the rate of convergence on main (as it currently stands).